roles/nova-data/tasks/main.yml

```diff
@@ -27,6 +27,17 @@
 
 - modprobe: name=nbd state=present
 
+- name: Turning off SMT mode for ppc64le
+  command: "ppc64_cpu --smt=off"
+  when: ansible_architecture == "ppc64le"
+
+- name: Adding SMT mode off in rc.local
+  lineinfile: 
+    dest: /etc/rc.local 
+    line: "ppc64_cpu --smt=off"
```

Comment:
We already have some facility for updating grub options. I'd suggest making a change that ties into that. https://github.com/blueboxgroup/ursula/blob/master/roles/common/tasks/kernel-tuning.yml